### PR TITLE
view now properly scroll to following

### PR DESCRIPTION
**Pull request by jankrzempek**
https://trello.com/c/posaRwe2/361-home-button-navigation

### CODE REVIEW

File: AirCasting/Dashboard/AirSectionPickerView.swift [hunk 1]
```diff
@@ -2,6 +2,7 @@
 //
 
 import SwiftUI
+import Combine
```
Comment (smialko) [inline AirCasting/Dashboard/AirSectionPickerView.swift:5]: Why did you add Combine? You don't need it :)

File: AirCasting/Dashboard/AirSectionPickerView.swift [hunk 2]
```diff
@@ -14,14 +15,16 @@ struct AirSectionPickerView: View {
                     ForEach(SelectedSection.allCases, id: \.self) { section in
                         Button(section.localizedString) {
                             withAnimation(.easeInOut(duration: 0.1)) {
-                                if section == .mobileDormant {
+                                if section == .mobileDormant || section == .fixed {
```
Comment (smialko) [inline AirCasting/Dashboard/AirSectionPickerView.swift:18]: Why? :P We don't need that, do we?

File: AirCasting/Dashboard/AirSectionPickerView.swift [hunk 3]
```diff
@@ -14,14 +15,16 @@ struct AirSectionPickerView: View {
...
                                     scrollReader.scrollTo(SelectedSection.fixed)
                                 } else if section == .mobileActive {
                                     scrollReader.scrollTo(SelectedSection.following)
                                 }
                                 selection = section
                             }
-                        }
+                        }.onChange(of: selection, perform: {
+                            $0 == .following ? scrollReader.scrollTo(SelectedSection.following) : nil
```
Comment (smialko) [inline AirCasting/Dashboard/AirSectionPickerView.swift:26]: Could you add animation here? It pops to the left and it's kinda harsh :D

**Review by smialko — COMMENTED** [review]
Good job ;D

File: AirCasting/Dashboard/AirSectionPickerView.swift (hunk 1 above)
Comment (jankrzempek) [reply AirCasting/Dashboard/AirSectionPickerView.swift:5]: Yeah, I've added it when I had the other conception 😅

File: AirCasting/Dashboard/AirSectionPickerView.swift (hunk 2 above)
Comment (jankrzempek) [reply AirCasting/Dashboard/AirSectionPickerView.swift:18]: You do, because without it for now: when you are at the leading edge (Following) and try to tap onto fixed but without moving the AirSection at all the 'Fixed' text will be cuted off

Comment (smialko) [reply AirCasting/Dashboard/AirSectionPickerView.swift:18]: oh, okey, didn't try that edgy edge case :D 
Nice!
